implementations/rust/ockam/ockam_api/src/cli_state/storage/spaces_repository_sql.rs

```diff
@@ -176,7 +167,11 @@ impl SpacesRepository for SpacesSqlxDatabase {
                 space.users = users;
 
                 // retrieve the subscription
-                space.subscription = self.query_subscription(&space.id).await?;
+                // FIXME: Copy&paste
```

Comment:
Why is this necessary for this PR?

Reply:
Because it can't work if max number of db connections is 1